src/Credentials/GCECredentials.php

```diff
@@ -424,7 +424,8 @@ public function fetchAuthToken(callable $httpHandler = null)
         $response = $this->getFromMetadata($httpHandler, $this->tokenUri);
 
         if ($this->targetAudience) {
-            return ['id_token' => $response];
+            $this->lastReceivedToken = ['id_token' => $response];
+            return $this->lastReceivedToken;
```

Comment:
nit: we can consolidate this to one line
```suggestion
            return $this->lastReceivedToken = ['id_token' => $response];
```

Reply:
Done